plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/LogPropertyPropertiesEditionPartForm.java

```diff
@@ -540,7 +540,7 @@ protected Composite createPropertyExpressionText(FormToolkit widgetFactory, Comp
 		Control propertyExpressionTextLabel = createDescription(parent,
 				EsbViewsRepository.LogProperty.Properties.propertyExpression,
 				EsbMessages.LogPropertyPropertiesEditionPart_PropertyExpressionLabel);
-		propertyExpressionText = widgetFactory.createText(parent, ""); //$NON-NLS-1$
+		propertyExpressionText = widgetFactory.createText(parent, "", 8); //$NON-NLS-1$
```

Comment:
Remove the comment.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804